yt/visualization/plot_window.py

```diff
@@ -240,6 +240,7 @@ def fget(self):
         def fset(self, value):
             self._frb = value
             self._data_valid = True
+            self._plot_valid = False
```

Comment:
I don't think this is used anywhere

Reply:
It is at line 751 of the same file.

Reply:
Ah I see, this is a good cleanup then. Sorry for not looking at the whole file, just the diff.